src/app/account/account-page/account-page.component.html

```diff
@@ -0,0 +1,35 @@
+<cs-list
+  [isOpen]="listService?.hasSelected()">
+  <cs-account-list-filter *ngIf="isAdmin()"
+    [domains]="domains"
+    [roles]="roles"
+    [roleTypes]="roleTypes"
+    [states]="states"
+    [groupings]="groupings"
+    [selectedDomainIds]="selectedDomainIds"
+    [selectedRoleNames]="selectedRoleNames"
+    [selectedRoleTypes]="selectedRoleTypes"
+    [selectedStates]="selectedStates"
+    [selectedGroupings]="selectedGroupings"
+    (onDomainsChange)="onDomainsChange.emit($event)"
+    (onRolesChange)="onRolesChange.emit($event)"
+    (onRoleTypesChange)="onRoleTypesChange.emit($event)"
+    (onStatesChange)="onStatesChange.emit($event)"
+    (onGroupingsChange)="onGroupingsChange.emit($event)"
+    cs-list
+  ></cs-account-list-filter>
+  <cs-top-bar  *ngIf="!isAdmin()"></cs-top-bar>
```

Comment:
let it be present for all the users but the content should be conditionally appear